src/main/java/loanbook/model/loan/Nric.java

```diff
@@ -9,7 +9,7 @@
 public class Nric implements Censor {
 
     public static final String MESSAGE_NRIC_CONSTRAINTS =
-            "NRIC should be Singapore issued. It may be blank. ";
+            "Invalid NRIC input. NRIC should consist of a prefix letter, followed by 7 letters and a checksum suffix.";
```

Comment:
I think by taking away the "NRIC should be Singapore issued" message, users might be confused why "S0000000A" isn't working since, as far as they're concerned, it matches the format specified. (They might not know what a "checksum suffix" is, or how it works)

Though, I'm not sure what a suitable replacement should be... 

Perhaps just "Invalid NRIC input. It should be a valid Singapore-issued NRIC number", or something.

Reply:
Resolved!